db/seeds.rb

```diff
@@ -85,16 +134,110 @@
+    "Acompañamiento", "Salud", "Tareas domésticas", "Tareas administrativas",
+    "Clases", "Ocio", "Asesoramiento", "Otro"
   ].each do |name|
     unless Category.with_name_translation(name).exists?
       Category.create { |c| c.name = name }
     end
   end
 end
 
+Post.find_or_create_by(id: 1) do |post|
+  post.title = "Ruby on Rails nivel principiante"
+  post.description = "Te enseño a trastear un poco sin cuidar el codigo, pero con mucho amor.
+  Aprenderas a ser un coder de verdad http://rubyonrails.org/"
+  post.type = "Offer"
+  post.category_id = 5
+  post.user_id = 1
+  post.tags = ["Rails", "Ruby", "programación"]
+  post.organization_id = 1
+end
+
+Post.find_or_create_by(id: 2) do |post|
+  post.title = "Cocina low cost"
+  post.description = "Si **no sabes que puedes comer** y no quieres gastar mucho dinero te enseño como
+  hacer una sopa de plantas silvestres encontradas entre las grietas del asfalto"
+  post.type = "Offer"
+  post.category_id = 7
+  post.user_id = 1
+  post.tags = ["Cocinar", "Cocina"]
+  post.organization_id = 1
+end
+
+Post.find_or_create_by(id: 3) do |post|
+  post.title = "Yoga para principiantes"
+  post.description = "Comparteixo les tècniques que he après i he practicat de respiració i meditació (una és inherent a l´altra, l´altra és inherent a l´una). En cada sessió: minuts de consciencia postural, cant de mantres, *breu escalfament**
+  (movilització de l´articulacions i la columna vertebral) i una sèrie de PRANAYAMA (respiració) / MEDITACIÓ. Sessions de 1h aprox. *CAPACITACIÓ INTERNACIONAL DE MESTRES DE KUNDALINI IOGA, NIVELL I d’acord Yogi Bhajan. per la “KRI” (KUNDALINI RESEARCH INSTITUT)"
+  post.type = "Offer"
+  post.category_id = 5
+  post.user_id = 2
+  post.tags = ["Yoga", "Estiramientos", "Respiración", "Meditación"]
+  post.organization_id = 1
+end
+
+Post.find_or_create_by(id: 4) do |post|
+  post.title = "English conversation"
+  post.description = "Our specially designed courses are for adults looking to improve their proficiency in English. Whether you want to improve your overall communication, take an English exam, or simply want to develop your spoken English skills, we have the right course for you. All successful students will receive a British Council certificate at the end of the course."
+  post.type = "Offer"
+  post.category_id = 5
+  post.user_id = 2
+  post.tags = ["Inglés", "English", "Conversación"]
+  post.organization_id = 1
+end
+
+Post.find_or_create_by(id: 5) do |post|
+  post.title = "Te enseño a escribir con Markdown"
+  post.description = 
+  "An h1 header
+  ============
+
+  ![Image of Yaktocat](https://octodex.github.com/images/yaktocat.png)
+
+
+  Paragraphs are separated by a blank line.
+
+  2nd paragraph. *Italic*, **bold**, and `monospace`. Itemized lists
+  look like:
+
+    * this one
+    * that one
+    * the other one
+
+  Note that --- not considering the asterisk --- the actual text
+  content starts at 4-columns in.
+
+  http://www.google.com 
```

Comment:
```ruby
post.description = <<-EOF
An h1 header
============

...
EOF
```
Using `<<-MARKER` will also remove from all lines a leading space that's equal to the fist line's leading space. So the indentation is kept and beautiful.